### ISSUE TITLE
example.pd -> AudioWorklet not supported, defaults to ScriptProccessorNode

### ISSUE THREAD

**Issue by CarloCattano** [chrisuehlinger/hvcc#2]
building with emsdk latest:
```bash 
emcc -v
emcc (Emscripten gcc/clang-like replacement + linker emulating GNU ld) 3.1.14 (4343cbec72b7db283ea3bda1adc6cb1811ae9a73)
clang version 15.0.0 (https://github.com/llvm/llvm-project 7effcbda49ba32991b8955821b8fdbd4f8f303e2)
Target: wasm32-unknown-emscripten
Thread model: posix
InstalledDir: /home/manjaro/emsdk/upstream/bin

uname -a
Linux manjaro-21 5.10.123-1-MANJARO #1 SMP PREEMPT Thu Jun 16 12:34:16 UTC 2022 x86_64 GNU/Linux
```
serving it with ```python -m http.server```

tested on chrome , firefox and brave

built example.pd from examples folder with : 
```bash
hvcc examples/example.pd -o example -n example -g js -v
```

![image](https://user-images.githubusercontent.com/17380530/175810797-054be6b0-2cc9-4c03-9217-fd2f8809ab5a.png)

[hvcc_example_build_output.txt](https://github.com/chrisuehlinger/hvcc/files/8986451/hvcc_example_build_output.txt)

Chrome v103 -> Audio worklet supported according to https://caniuse.com/?search=audioworklet
Firefox v101 -> ""

**Comment by dromer** [comment]
For me this works in FF v101.0.1

**Comment by CarloCattano** [comment]
It works for me too but it defaults to an AudioScriptProcessor instead of a worklet. Can you check the console and see if its your case too @dromer?

**Comment by dromer** [comment]
@CarloCattano you are right: `heavy: AudioWorklet not supported, reverting to ScriptProcessorNode`

However this is when serving with `python -m http.server`.
If I scp it to my webserver running nginx then it does work: https://mrtoasted.com/~dreamer/hvcctest/bla/

I'm happy that the fallback works, though.

**Comment by CarloCattano** [comment]
Interesting an unexpected , then perhaps its not an issue after all and its just a perk about python serving the files and creating some weird issue.

**Comment by CarloCattano** [comment]
Just tried with rust´s simple-http-server and it works as it should , might be something about CORS or who knows , python http.server its very simplistic so makes sense.
Consider it closed perhaps @chrisuehlinger

**Comment by chrisuehlinger** [comment]
The plot thickens... Tonight I got my branch up and running on Windows (so essentially a fresh install for me) and was able to compile and run example.pd via `python -m http.server` with no console warnings or errors.

It's pretty weird that both of you ran into this and I didn't. I'll try on my Mac as well and see what happens.
